nuxeo-services/nuxeo-platform-directory/nuxeo-platform-directory-test/src/main/java/org/nuxeo/directory/test/DirectoryConfiguration.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * (C) Copyright 2017 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Funsho David
+ *
+ */
+
+package org.nuxeo.directory.test;
+
+import org.nuxeo.ecm.core.test.StorageConfiguration;
+import org.nuxeo.runtime.test.runner.FeaturesRunner;
+import org.nuxeo.runtime.test.runner.RuntimeFeature;
+import org.nuxeo.runtime.test.runner.RuntimeHarness;
+
+
+/**
+ * Description of the specific capabilities of a directory for tests, and helper methods.
+ *
+ * @since 9.2
+ */
+public class DirectoryConfiguration {
+
+    public static final String DIRECTORY_PROPERTY = "nuxeo.test.directory";
+
+    public static final String DIRECTORY_VCS = "vcs";
```

Comment:
`DIRECTORY_SQL = "sql"` please as it's the final storage backend that's important, now how we implement access to it.